### DISCUSSION TITLE
How to scrape data from a logged in user's for you page

### DISCUSSION THREAD

**Discussion by lakshman1997** [davidteather/TikTok-Api#1081] [Q&A]
Hi All,

I wanted to know can I use the trending API to get the data from a logged in user's For You page. I wanted to know how can I use the session and the user-id of a user to extract this data. Please help.

**Comment by eliich** [comment]
Hello just want to let you know that is not possible in this codebase, to do that you would need to create your own wrapper API and 
1. adjust the signature parameters(through playwright) and cookies when making requests to being logged in
2. accept cookies(which is the reason this and most packages don't allow as scraping content is against TOS).
I recommend you to take a look at David's reponse as it explains it better: https://github.com/davidteather/TikTok-Api/discussions/428
